queue/src/main/java/org/killbill/queue/DBBackedQueue.java

```diff
@@ -783,10 +783,11 @@ public T apply(@Nullable final T entry) {
                         entry.setCreatedDate(now);
                         entry.setProcessingState(PersistentQueueEntryLifecycleState.AVAILABLE);
                         entry.setCreatingOwner(CreatorName.get());
-                        insertEntryFromTransaction(transactional, entry);
                     }
 
-                    log.warn(String.format( "{} {} entries were reaped by {}",DB_QUEUE_LOG_ID ,entriesLeftBehind.size(), CreatorName.get()));
+                    transactional.insertEntries(entriesLeftBehind, config.getTableName());
```

Comment:
@wwjbatista I think for this PR (`STICKY_POLLING`), bypassing the logic from `insertEntryFromTransaction` going straight to the DB is fine, although we should at least update the stats (`totalInsert.inc()`).

@sbrossie But I don't think this is what we want for the full implementation (i.e. considering `STICKY_EVENTS` https://github.com/killbill/killbill-commons/issues/30) as we wouldn't re-insert the `ids` into the inflight queue (unless we want to solely rely on the orphan polling mechanism?).